NDTensors/test/ITensors/TestITensorDMRG/dmrg.jl

```diff
@@ -0,0 +1,29 @@
+using Test
+using ITensors
+using NDTensors
+using Random
+
+function test_dmrg(elt, N::Integer, dev::Function)
+  sites = siteinds("S=1/2", N)
+
+  os = OpSum()
+  for j in 1:(N - 1)
+    os += "Sz", j, "Sz", j + 1
+    os += 0.5, "S+", j, "S-", j + 1
+    os += 0.5, "S-", j, "S+", j + 1
+  end
+
+  Random.seed!(1234)
+  psi0 = dev(randomMPS(Float64, sites;))
+  H = dev(MPO(elt, os, sites; linkdims=4))
```

Comment:
```suggestion
  psi0 = dev(randomMPS(Float64, sites; linkdims=4))
  H = dev(MPO(elt, os, sites))
```
The link dimensions of the MPO are determined by the model (encoded in the `OpSum`) so it doesn't make sense to try to set them. In principle it should have errored since that isn't a supported keyword argument of `MPO`.